### ISSUE TITLE
Recent update blacks out the UI and makes every single option impossible to read

### ISSUE THREAD

**Issue by Jan-Blasiak** [CleverRaven/Cataclysm-DDA#52046]
### Describe the bug

After updating the game the the UI blacks out  and makes every single option impossible to read

### Steps To Reproduce

launch the game, and this is what you see:

![Zrzut ekranu (460)](https://user-images.githubusercontent.com/61629226/135715315-b184e852-dd82-4121-83d9-e513d96cb6d8.png)

### Expected behavior

Normal menu

### Versions and configuration

v: 0436c6d (tiles)

**Comment by BrettDong** [comment]
Did you try the latest experimental?

**Comment by Theawesomeboophis** [comment]
I can confirm that this is happening to me too.

**Comment by BrettDong** [comment]
This is what I get from https://github.com/CleverRaven/Cataclysm-DDA/releases/tag/cdda-experimental-2021-10-02-1005

<img width="821" src="https://user-images.githubusercontent.com/21075502/135719830-a9996dfe-42ab-4761-8cea-2d7264929157.png">

**Comment by Theawesomeboophis** [comment]
That's strange, are you using windows?

**Comment by BrettDong** [comment]
> That's strange, are you using windows?

Yes, I tested on both Windows 7 and Windows 8.1 test environments.

**Comment by Theawesomeboophis** [comment]
Maybe it's a launcher problem? I'll test on catapult later to see if it works there.

**Comment by Zireael07** [comment]
Or they have some old config files pointing to the ttf version of Terminus? Although in that case, the fallback font should come on...

**Comment by Theawesomeboophis** [comment]
That might be my case as well.

**Comment by ZhilkinSerg** [comment]
Duplicate of https://github.com/CleverRaven/Cataclysm-DDA/issues/50149

**Comment by Maleclypse** [comment]
> Duplicate of #50149

Is it really a duplicate if it's happening on windows now or is it a duplicate in that they probably have the same solution?

**Comment by BrettDong** [comment]
The original issue should be edited to indicate the problem exists regardless of operating system.

**Comment by BrettDong** [comment]
But the font has been replaced by a working one in #52017 so how it's possible to happen in the latest experimental today?

**Comment by ZhilkinSerg** [comment]
> But the font has been replaced by a working one in #52017 so how it's possible to happen in the latest experimental today?

I assume they are still using non-working font as we did not reset font.config entries.

**Comment by BrettDong** [comment]
> I assume they are still using non-working font as we did not reset font.config entries.

But the problematic TTF font was deleted in #52017, so how the old font can possibly be loaded?

**Comment by ZhilkinSerg** [comment]
> > I assume they are still using non-working font as we did not reset font.config entries.
> 
> But the problematic TTF font was deleted in #52017, so how the old font can possibly be loaded?

Old font was not deleted during update with launcher or if game was unzipped over existing installation.

**Comment by Theawesomeboophis** [comment]
So deleting old font from fonts.json should fix it?

**Comment by morthein** [comment]
hey, i just updated to the latest and got this too. Everything is pitch black, I can still navigate to a saved game from memory, but none of the game tiles show up

**Comment by morthein** [comment]
I went ahead and installed the game in a brand new directory and it shows up just fine with the latest update.

**Comment by mpyn** [comment]
> This is what I get from https://github.com/CleverRaven/Cataclysm-DDA/releases/tag/cdda-experimental-2021-10-02-1005
> 
> <img alt="" width="821" src="https://user-images.githubusercontent.com/21075502/135719830-a9996dfe-42ab-4761-8cea-2d7264929157.png">

Sorry to write here. But the new font looks so awful. How to set it up to be readable and beautiful, like with the old font?

**Comment by mythosmod** [comment]
Seems like an issue with the CDDA Game Launcher, had same issue updating to latest version - resolved by deleting fonts.json from the cdda/config folder within the launcher folder structure which allowed the new font to be loaded on launch (file recreated automatically)

AFAIK from watching the update logs on the launcher it backs up and restores the config folder as part of the update so I would suspect this is the root cause if the font has been changed in the newer versions.

**Comment by AcidAntOnAMinefield** [comment]
After deleting fonts.json and updating the game, the text is visible once again. 
But good grief it's borderline unreadable. 
So I just deleted the entries for the VecTerminus font on the new Json and left unifont as the only one. Much better on the eye.
Which makes me ask: why isn't there an option in-game to switch between fonts? Call me blind if there is one, but I can't for the life of me spot it.

**Comment by BrettDong** [comment]
> But the new font looks so awful. How to set it up to be readable and beautiful, like with the old font?

I have tried a lot of fonts, and among those working fonts that does not cause black screen, this one is the least ugliest.

If you can find a functioning font, and also looks nice, feel free to propose a pull request.

**Comment by mpyn** [comment]
> > But the new font looks so awful. How to set it up to be readable and beautiful, like with the old font?
> 
> I have tried a lot of fonts, and among those working fonts that does not cause black screen, this one is the least ugliest.
> 
> If you can find a functional font that also looks nice, feel free to propose a pull request.

If delete the path to the font in fonts.json, it looks much nicer :) There are no additional fonts installed in my Windows, most likely it uses some kind of system font

![cddaf](https://user-images.githubusercontent.com/5890610/135747672-cfb64624-8b5b-4448-8cd9-33798721e4b5.jpg)
.

**Comment by Zireael07** [comment]
Most likely without deleting the old, buggy path, it was falling back to unifont and therefore looking ugly.

**Comment by mpyn** [comment]
> Most likely without deleting the old, buggy path, it was falling back to unifont and therefore looking ugly.

With a new right path, it doesn't look good

![cataf3](https://user-images.githubusercontent.com/5890610/135748137-fe5fa26b-0c4e-4de8-8572-920abda80ec5.jpg)

**Comment by mpyn** [comment]
oops, i was wrong, the game uses a unifont.ttf from the game folder "fonts"

**Comment by hafnhaf** [comment]
> > > But the new font looks so awful. How to set it up to be readable and beautiful, like with the old font?
> > 
> > 
> > I have tried a lot of fonts, and among those working fonts that does not cause black screen, this one is the least ugliest.
> > If you can find a functional font that also looks nice, feel free to propose a pull request.
> 
> If delete the path to the font in fonts.json, it looks much nicer :) There are no additional fonts installed in my Windows, most likely it uses some kind of system font
> 
> ![cddaf](https://user-images.githubusercontent.com/5890610/135747672-cfb64624-8b5b-4448-8cd9-33798721e4b5.jpg) ![cataf2](https://user-images.githubusercontent.com/5890610/135747979-12248cc8-8e8d-48ba-a217-556a2ec9957f.jpg)

This worked perfectly.

**Comment by BrettDong** [comment]
I reverted FreeType library back to 2.10.4 in my custom MXE toolchain (https://github.com/BrettDong/MXE-GCC/commit/600c353eb52956e7b1d9faf0784e57dc8c830784). The next experimental release should be able to display the old Terminus.ttf font correctly.

**Comment by ZhilkinSerg** [comment]
We should restore deleted fonts btw.

**Comment by BrettDong** [comment]
However Linux and Mac users will still have problem with Terminus TTF unless they can find a way to install the old version of FreeType.

**Comment by Midaychi** [comment]
On a side note, mplus-2m-medium.otf doesn't have the same edgy angles but it works visually.
Basis33 is very readable too but its a little small

**Comment by ZhilkinSerg** [comment]
We should restore deleted fonts btw.

> However Linux and Mac users will still have problem with Terminus TTF unless they can find a way to install the old version of FreeType.

Maybe we shall apply different default fonts depending on linked/compiled SDL2_ttf/FreeType version?
